cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/ProductDTO.java

```diff
@@ -408,11 +416,29 @@ public void setTags( Set<ProductTagDTO> tags )
         this.tags = tags;
     }
 
+    public Set<String> getSyndicationEndpoints()
+    {
+        return syndicationEndpoints;
+    }
+
+    public void setSyndicationEndpoints( Set<String> syndicationEndpoints )
+    {
+        this.syndicationEndpoints = syndicationEndpoints;
+    }
+
+    /**
+     * @deprecated by {@link #getSyndicationEndpoints()}
+     */
+    @Deprecated
     public String getSyndicatedEndpoint()
     {
         return syndicatedEndpoint;
     }
 
+    /**
+     * @deprecated by {@link #setSyndicationEndpoints(Set)}
+     */
+    @Deprecated
     public void setSyndicatedEndpoint( String syndicatedEndpoint )
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)